packages/fxa-content-server/app/scripts/lib/app-start.js

```diff
@@ -428,6 +428,18 @@ Start.prototype = {
   },
 
   createView (Constructor, options = {}) {
+    const {
+      subscriptions: {
```

Comment:
Please remove the development config from code and rely upon the [passed in config from get-index.js](https://github.com/mozilla/fxa/pull/940/files#diff-da283abf354df26a8bebd2d236b18f39R55).